modules/openxr/openxr_api.cpp

```diff
@@ -2072,6 +2073,25 @@ void OpenXRAPI::set_foveation_dynamic(bool p_foveation_dynamic) {
 	}
 }
 
+Size2 OpenXRAPI::get_play_space_bounds() const {
+	Size2 ret;
+
+	ERR_FAIL_COND_V(session == XR_NULL_HANDLE, Size2());
+
+	XrExtent2Df extends;
```

Comment:
```suggestion
	XrExtent2Df extents;
```
Assuming this was meant, "the size", as opposed to "instances extending"

Reply:
Yeah, my broken dyslectic English... it even spells it correctly in the type name and I don't see it... :P